### DISCUSSION TITLE
No visualization?

### DISCUSSION THREAD

**Discussion by huisky2000** [InteractiveComputerGraphics/SPlisHSPlasH#138] [Q&A] [answered]
Hi,
I successfully compiled the code in VS2019. 
I is strange that I can only visualize some of the scenes.

For example
.\SPHSimulator_d.exe ..\data\Scenes\Beam_2D.json   --- OK!
.\SPHSimulator_d.exe ..\data\Scenes\AnimatedBody_2D.json --- OK!

but 
.\SPHSimulator_d.exe ..\data\Scenes\Beam.json --- NOT OK!
.\SPHSimulator_d.exe ..\data\Scenes\DamBreakModel_2D.json  --- NOT OK!
and more. 

The same effects, I can see the GUI initilized, but not responding for a long time. Any ideas?
I doubt is it due to my graphaic card probelm?

Example outputs for failed results are as follows:

.\SPHSimulator_d.exe ..\data\Scenes\Beam.json
SPlisHSPlasH version: 2.11.1
Output directory: C:/Arbeid/SPlisHSPlasH-master/bin/output/Beam
Load scene file: C:/Arbeid/SPlisHSPlasH-master/data/Scenes/Beam.json
Initialize static boundary simulation
Initialize fluid particles
Number of fluid particles: 8982
OpenGL version 4.6
Vendor: Intel
Renderer: Intel(R) UHD Graphics
Version: 4.6.0 - Build 30.0.100.9864
Object 0 - fixed particles: 1133

and for the other case
.\SPHSimulator_d.exe ..\data\Scenes\DamBreakModel_2D.json
SPlisHSPlasH version: 2.11.1
Output directory: C:/Arbeid/SPlisHSPlasH-master/bin/output/DamBreakModel_2D
Load scene file: C:/Arbeid/SPlisHSPlasH-master/data/Scenes/DamBreakModel_2D.json
Initialize static boundary simulation
Initialize fluid particles
Number of fluid particles: 841
OpenGL version 4.6
Vendor: Intel
Renderer: Intel(R) UHD Graphics
Version: 4.6.0 - Build 30.0.100.9864
Loading C:/Arbeid/SPlisHSPlasH-master/data/models/UnitBox.obj
Number of triangles: 12
Number of vertices: 8
Domain - min: -2.8, -2.8, -1.3
Domain - max: 2.8, 2.8, 1.3
Set SDF resolution: 30, 30, 20
Generate SDF

Rgds
H. L

**Comment by janbender** [comment, marked as answer]
I have no experience with the Intel OpenGL driver. Could this be a problem?
Did you also try with the release version?

**Comment by huisky2000** [comment]
Yes, it is working with the 'release' mode. Thanks!
